Form-Controls/index.html

```diff
@@ -13,16 +13,40 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
+    <form>
+      <label for="Fullname">Full Name:</label>
```

Comment:
When you ran Lighthouse, what did it say about this line?

https://dequeuniversity.com/rules/axe/4.4/label